src/quo2/components/messages/system_message.cljs

```diff
@@ -4,41 +4,13 @@
             [quo2.components.markdown.text :as text]
             [quo2.components.messages.author.view :as author]
             [quo2.foundations.colors :as colors]
-            [quo2.theme :as theme]
             [react-native.core :as rn]
-            [react-native.reanimated :as reanimated]))
-
-(def themes-landed
-  {:pinned  colors/primary-50-opa-5
-   :added   colors/primary-50-opa-5
-   :deleted colors/danger-50-opa-5})
+            [utils.i18n :as i18n]
+            [quo2.theme :as quo.theme]))
 
 (def themes
```

Comment:
maybe "text-color" is a better name here?
also consider doing 
```
(defn text-color [theme]
  (colors/theme-colors colors/neutral-100 colors/white theme))
 ```
 as this is how we are mostly using this across the codebase now